gsd/hoomd.py

```diff
@@ -978,13 +980,42 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.file.close()
 
 
+def open_handle(name, mode='rb', schema_version=(1, 4)):
```

Comment:
Do we need this? Or should we document the `file` attribute of HOOMDTrajectory and get the handle like this?
```
traj = gsd.hoomd.open(...)
handle = traj.file()
```

Reply:
We could document the `file` attribute, in which case I would suggest making it a property.

I think accessing the file directory is definitely useful when you want fast access to specific chunks without the overhead of full file reading. I've mentioned before that I use that feature for fast label evaluation in signac-flow status updates. The only way I know what arguments to pass to `fl.open` for application, schema, and schema_version by looking at `gsd.hoomd.open` (and I only later discovered that some of these arguments were optional), so I think exposing the raw open file handle would be useful. However, I have no problem with recommending that it be accessed through a property of the trajectory.

Reply:
Yes, a property would be best for `HOOMDTrajectory.file`. Exposing this would remove the need for `hoomd.open_handle` and reduce duplication of APIs: `fl.open`, `hoomd.open_handle`, and `hoomd.open`.

To deal with the performance issue you mention, we could refactor the `HOOMDTrajectory` object to lazily load data only when requested. That would require a significant amount of effort and is beyond the scope of this PR. If anyone is interested, feel free to make an issue.

Reply:
I agree that refactoring is out of scope, I was simply stating that in current master even making the file handle is nontrivial because it requires knowing the appropriate values for the the schema, schema_version, and application arguments. Exposing the `file` as a property would largely address that issue.